draft-ietf-tls-tls13.md

```diff
@@ -3131,7 +3185,8 @@ aspect of the certificate chain was unacceptable (e.g., it was not signed by a
 known, trusted CA), the server MAY at its discretion either continue the
 handshake (considering the client unauthenticated) or abort the handshake.
 
-Any endpoint receiving any certificate signed using any signature algorithm
+Any endpoint receiving any certificate that must be validated and is
+signed using any signature algorithm
```

Comment:
I was semi-remembering discussion about roots that were self-signed with md5, but do not wish to reopen the discussion.
@seanturner do you remember what I might be thinking about?